libs/prelude/Prelude.idr

```diff
@@ -37,6 +37,8 @@ import public Language.Reflection.Errors
 %default total
 
 -- Things that can't be elsewhere for import cycle reasons
+%runElab (defineFunction $ DefineFun `{void} [])
```

Comment:
Sorry, I'm really busy right now! Yes, I think that having comments pointing between the two _and_ summarizing the dependency issue should be ok. After all, `void` doesn't compute :-)